bin/underreact.js

```diff
@@ -106,12 +147,16 @@ function defineServeStatic(y) {
 }
 
 function runServeStatic(argv) {
-  config('serve-static', argv)
+  const serveStatic = require('../commands/serve-static');
+  const config = require('../lib/config');
```

Comment:
If every command ends up requiring `config`, why not require it at the top of the file?

Reply:
The problem with requiring it at the top of file would mean that it wouldn't get the necessary `env` variable setup, leading to unpredictability. Hence I am trying to defer most of the `require`s until the [middleware](https://github.com/mapbox/underreact/pull/21/files#diff-ecf66e53d7f5563584528930a084b7f4R15) has executed, in order to setup the necessary `env` vars.

Reply:
We would want our env variables to be set before anything else executes and to make that happen we also need a normalised `argv` coming from `yargs`, that is why you would the env vars are being set inside the yargs middleware. I have left a comment about that [here](https://github.com/mapbox/underreact/blob/b14fed5edf90cab492280ed046d37524511efb1e/bin/underreact.js#L10)

Reply:
Oh, huh 👍